pyinfraformat/plots/holes.py

```diff
@@ -0,0 +1,114 @@
+"""Plots for single hole
+different plots for each hole type
+"""
+import pandas as pd
+import matplotlib.pyplot as plt
+import mpld3
+
+
+def plot_po(one_survey, return_value="fig"):
+    """Plot a PO, Porakonekairaus
+    returns a mpld3 html or matplotlib fig"""
+    df = pd.DataFrame(one_survey.survey.data)
+    if "Soil type" in df.columns:  # pylint: disable=unsupported-membership-test
+        soils = df.dropna(subset=["Soil type"])
+    else:
+        soils = None
+    df = pd.concat((df, df)).sort_values(by=["Depth (m)", "linenumber"])
+    df["Depth (m)"] = df["Depth (m)"].shift(1)
+
+    fig, (left, right) = plt.subplots(
+        1, 2, sharey=True, gridspec_kw={"wspace": 0, "width_ratios": [2, 2]}
+    )
+    fig.set_figwidth(4)
+    left.plot(df["Time (s)"], df["Depth (m)"], c="k")
+    left.invert_yaxis()
+    left.spines["top"].set_visible(False)
+    left.spines["left"].set_visible(False)
+    left.get_yaxis().set_visible(False)
+    left.set_xlim([110, 0])
+    plt.setp(left.get_yticklabels(), visible=False)
```

Comment:
plt.yticks([]) ?

Or do we actually need the ticks

Reply:
Two figures with own y scales, we need only one in the middle